src/js_stream.cc

```diff
@@ -159,23 +161,29 @@ void JSStream::DoRead(const FunctionCallbackInfo<Value>& args) {
 
 
 void JSStream::DoAfterWrite(const FunctionCallbackInfo<Value>& args) {
-  JSStream* wrap = Unwrap<JSStream>(args.Holder());
-  WriteWrap* w = Unwrap<WriteWrap>(args[0].As<Object>());
+  JSStream* wrap;
+  // TODO(trevnorris): Verify that args[0] is actually an Object.
```

Comment:
I'd add a CHECK.  If it's not an object, there's a logic error in node somewhere, right?